plugin.video.vrt.nu/resources/lib/kodiwrappers/kodiwrapper.py

```diff
@@ -72,6 +74,9 @@ def get_setting(self, setting_id ):
     def open_settings(self):
         self._addon.openSettings()
 
+    def has_inputstream_adaptive_installed(self):
```

Comment:
you cannot install the addon without inputstream adaptive since its a prerequisite in the addon.xml

Reply:
I know, but you can uninstall inputstream.adaptive in Kodi when the VRT NU addon is disabled. I added this in the first place to preserve support for the non-drm hls streams. Just to be prepared for a future worst case scenario when all other streams fail to play.

Reply:
Will merge but add a comment, else its confusing